lib/components/af.js

```diff
@@ -66,10 +66,10 @@ af.send = function (srcEp, dstEp, cId, rawPayload, opt, callback) {
 
     areqTimeout = opt.hasOwnProperty('timeout') ? opt.timeout : undefined;
 
-    senderEp = srcEp.isLocal() ? srcEp : controller.getCoord().getDelegator(profId);
+    senderEp = srcEp.isLocal() ? srcEp : controller._coord.getDelegator(profId);
```

Comment:
In javascript it's a convention to not use properties with `_` externally. Either we should put back `getCoord` or we should rename `_coord` to `coord`.

Reply:
It's an intentional smell.
They claim it's private, but allow getting it directly.
I made the minimum change that didn't require me making the call of making it public.
Converting the projects to TypeScript which makes a distinction between `private` and `readonly` (and generally honors them) will bring more clarity.